test/e2e/pvc_test.go

```diff
@@ -189,7 +191,7 @@ func TestDefaultPVC(t *testing.T) {
 }
 
 func TestCustomRWXPVC(t *testing.T) {
-	claimName := "test-custom-rwx-pvc"
+	claimName := uuid.New().String()
```

Comment:
Add some prefix so we can know where it came from in case of problems.

Reply:
done